index.html

```diff
@@ -1,93 +1,138 @@
 <!DOCTYPE html>
 <html lang="ko">
-<head>
-  <meta charset="utf-8">
-  <title>판다마켓</title>
-  <link rel="stylesheet" href="style.css">
-</head>
-<body>
-  <main>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>판다마켓</title>
+    <link rel="icon" href="images/logo/favicon.ico" />
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css"
+    />
+    <link rel="stylesheet" href="styles/style.css" />
+  </head>
+
+  <body>
     <header>
-      <div id="top">
-          <a href="/" target="_blank">
-            <img id="pmarket" src="images/logopanda.png" alt="판다마켓">
-          </a>
-          <a href="login.html" target="_blank">
-            <img src="images/login2.png" alt="로그인">
-          </a>
-      </div>
-  </header>
-  <section>
-    <div id="tlogos">
-      <div id="fimages">
-        <h1 class="word1">일상의 모든 물건을<br>거래해보세요</h1>
-        <a href="/items" target="_blank">
-          <img class="click" src="images/shopping.png" alt="구경하러 가기">
-        </a>
-      </div>
-      <img id="gomdori"
-      src="images/Img_home_top.png" alt="곰돌이">
-    </div>
-  </section>
-      <div class="mainsqua">
-        <div class="squa1">
-          <img src="images/Img_home_01.png" alt="옷">
-          <div class="ments1">
-            <p>Hot item</p>
-            <h1 class="word1">인기 상품을<br>확인해보세요</h1>
-            <h2 class="mword2">가장 HOT한 중고거래 물품을<br>판다 마켓에서 확인해 보세요</h2>
+      <a href="/"
+        ><img
+          src="images/logo/panda-image.png"
+          alt="판다마켓 홈"
+          width="153"
+      /></a>
+      <a href="login.html" id="loginLinkButton" class="button">로그인</a>
+    </header>
+
+    <main>
+      <section id="hero" class="banner">
+        <div class="wrapper">
+          <div id="banner-content">
+          <h1>
+            &nbsp; 일상의 모든 물건을
```

Comment:
P2:
불필요한 공백이므로 없는 것이 더 좋을 것 같습니다.

```suggestion
            일상의 모든 물건을
```